server/src/main/java/org/apache/druid/catalog/model/table/DatasourceDefn.java

```diff
@@ -68,16 +68,6 @@ public class DatasourceDefn extends TableDefn
    */
   public static final String HIDDEN_COLUMNS_PROPERTY = "hiddenColumns";
 
-  /**
-   * By default: columns are optional hints. If a datasource has columns defined,
```

Comment:
Why does this description no longer apply?

Reply:
I'm not sure why this was removed in Paul's original PR